Week6/Day2/Blahgger/Blahgger/Views/Account/Login.cshtml

```diff
@@ -0,0 +1,43 @@
+﻿@model Blahgger.Models.User
+
+@{
+    ViewBag.Title = "Login";
+}
+
+<h2>Login</h2>
+
+@using (Html.BeginForm())
+{
+    @Html.AntiForgeryToken()
```

Comment:
Don't forget to add a `ValidateAntiForgeryToken` attribute to the POST action method.